### ISSUE TITLE
Advancing medical imaging informatics by deep learning-based domain adaptation

### ISSUE THREAD

**Issue by standing-o** [standing-o/Machine_Learning_Paper_Review#23]
## Advancing medical imaging informatics by deep learning-based domain adaptation
- Authors: Choudhary, Anirudh and Tong, Li and Zhu, Yuanda and Wang, May D
- Journal: Yearbook of medical informatics
- Year: 2020
- Link: https://www.thieme-connect.com/products/ejournals/pdf/10.1055/s-0040-1702009.pdf

### Summary
- Getting largescale labeled data remains a challenge, and multi-center datasets suffer from heterogeneity due to patient diversity and varying imaging protocols. 
- `Domain adaptation (DA)` has been developed to transfer the knowledge from a labeled data domain to a related but unlabeled domain in either image space or feature space.
- `DA` is a type of transfer learning (TL) that can improve the performance of models when applied to multiple different datasets. 
- We discussed domain transformation (DT) and latent feature-space transformation (LFST).

### Introduction
- While multicenter datasets can increase the amount of annotated data, these datasets suffer from heterogeneity due to varying
hospital procedures and diverse patient populations.
  - Due to a distribution shift (also known as domain-shift) between the available training dataset and the dataset encountered in clinical practice, pre-trained models trained by one dataset may fail for another dataset. 

#### What is Transfer Learning and `Domain Adaptation`
- Transfer learning (TL) applies knowledge learned from one domain and one task to another related domain and/or another task.
- For medical imaging, a domain usually refers to images or features, while the task refers to segmentation, classification, etc.
- If both source $(D_s)$ and target domains $(D_T)$ are similar, then $D_s$ and $D_T$ can use the same ML model for similar tasks $(T_S ~ T_T)$.
  - If $D_s \neq D_T$ or $T_S \neq T_T$, the ML model trained on the source domain might habve decreased performance on the target domain $(D_T)$.
  1. Inductive TL requires some labeled data. While the two domains may or may not differ $(D_s ~ D_T)$ or $(D_s \neq D_T)$, the target and source tasks are different $(T_S \neq T_T)$.
  ➔ ex. Lung tumor detection across X-Ray and computed tomography images.
  2. Transductive TL requires labeled source data and unlabeled target data with related domains $(D_s ~ D_T)$ and same tasks
$(D_s = D_T)$, while the marginal probability distributions differ $(p(X_S \neq X_T))$.
  ➔ ex. Lung tumor detection across X-Ray and computed tomography images.
  3. Unsupervised TL does not require labeled data in any domain and has different tasks $(T_S \neq T_T)$.
  ➔ ex. Classifying cancer for different anatomies using unlabeled histology images.
- `Domain Adaptation (DA)` is a **2. transductive TL** approach that aims to transfer knowledge across domains by learning domain-invariant transformations, which align the domain distributions.

    <img src="https://github.com/standing-o/Machine_Learning_Paper_Review/assets/57218700/a1f0a0c5-b95c-41ee-a32a-f18143a40627" width=90%>

#### Using Domain Adaptation to improve model training in medical imaging
- Cross-modal `DA` transfers labels between distinct, but somewhat related, image modalities.
  - Single-modality `DA` adapts different image distributions within the same modality.
- `DA` can mitigate the lack of well-annotated data by augmenting target domain data, either by generating synthetic labeled images from source images or aligning source and target image features and training a task network on them.
- Through `DA`, annotated MRI scans from historical subjects can be combined with CT to reduce the number of image acquisitions needed.
- H&E stained images are widely available, while IHC images, which clearly highlight nuclei via specific biomarkers, are not.
  - `DA` methods can translate multi-stained H&E-stained images to the IHC domain, making nuclei detection easier.
- `Domain Adaptation` methodologies
    <img src="https://github.com/standing-o/Machine_Learning_Paper_Review/assets/57218700/6a2433a8-fb10-4210-ba40-768eef9beea2" width=90%>

#### Challenge of Dataset Variations
- Pathology images have stain variations whiel MRIs are susceptible to varing magnetic fields and contrast agents.
- Such intra- or inter-dataset variations cause the tarining and test dataaset to have different distributions, resulting in a domian-shift which impacts model generalization.
- `DA` methods try to minimize the dataset variation, while retaining the distinguishing aspects for task classifier, and have been shown to generalize well in image sesgmentation tasks for multiple modalities.

### Deep Learning-based `Domain Adaptation`
- Two families of DA approaches for medical imaging:
-
